consensus/src/creation/mod.rs

```diff
@@ -55,16 +55,23 @@ async fn create_unit<H: Hasher>(
     round: Round,
     creator: &mut Creator<H>,
     create_lag: &DelaySchedule,
-    mut can_create: bool,
+    mut try_create: bool,
     incoming_parents: &mut Receiver<Unit<H>>,
     mut exit: &mut oneshot::Receiver<()>,
 ) -> Result<(PreUnit<H>, Vec<H::Hash>), ()> {
-    let mut delay = Delay::new(create_lag(round.into())).fuse();
+    const THIRTY_MINUTES: Duration = Duration::from_secs(30 * 60);
+
+    let initial_delay = match try_create {
+        false => create_lag(round.into()),
+        true => THIRTY_MINUTES,
+    };
+    let mut delay = Delay::new(initial_delay).fuse();
     loop {
-        if can_create {
+        if try_create {
             if let Some(result) = creator.create_unit(round) {
                 return Ok(result);
             }
+            debug!(target: "AlephBFT-creator", "Creator unable to create a new unit at round {:?}.", round);
```

Comment:
If you follow @kostekIV 's suggestion about errors, this would be a very nice place to print them.